helm-charts/common/tgi/templates/deployment.yaml

```diff
@@ -98,7 +98,7 @@ spec:
               name: shm
             - mountPath: /tmp
               name: tmp
-            - mountPath: /usr/src/out
+            - mountPath: {{ .Values.tokenizerMountPath | default /usr/src/out }}
```

Comment:
Why we need to make this user configurable if all the 3 versions of tgi image share the same value?